contracts/Commitment/Commitment.sol

```diff
@@ -349,6 +340,7 @@ contract Commitment is
         constant
         returns (uint256)
     {
+        require(amount < 2**123);
```

Comment:
Where is the number `2**123` coming from?

Reply:
There is explanations two lines above. I'll added a small details there, now as follows
```
/// @notice Considering the max possible ETH_EUR_FRACTION value (10**18*10**4 == ~2**73), the max
    ///       amount of ETH (not wei) that is safe to be passed as the argument
    ///       is ~10**(54 - 18) (~2**123).
```